src/tests/end2end/ComputeCopyStorageBufferTests.cpp

```diff
@@ -109,7 +109,9 @@ TEST_P(ComputeCopyStorageBufferTests, BasicTest) {
 }
 
 // Test that a slightly-less-trivial compute-shader memcpy implementation works.
-TEST_P(ComputeCopyStorageBufferTests, StructTest) {
+//
+// TODO(kainino@chromium.org): Fails on D3D12 backend. Probably due to a limitation in SPIRV-Cross?
+TEST_P(ComputeCopyStorageBufferTests, SizedArrayOfStruct) {
```

Comment:
If it fails on some backends, can we skip it like we skip some tests in BufferTests?